appium/webdriver/webdriver.py

```diff
@@ -440,6 +439,8 @@ def _addCommands(self):
             ('POST', '/session/$sessionId/touch/multi/perform')
         self.command_executor._commands[Command.SET_IMMEDIATE_VALUE] = \
             ('POST', '/session/$sessionId/appium/element/$id/value')
+        self.command_executor._commands[Command.GET_EVENTS] = \
```

Comment:
there is also an endpoint to add events. I think it makes sense to extract these two into a separate mixin